openedx/core/djangoapps/course_groups/management/commands/remove_users_from_multiple_cohorts.py

```diff
@@ -0,0 +1,56 @@
+"""
+Script for removing users with multiple cohorts of a course from cohorts
+to ensure user's uniqueness for a course cohorts
+"""
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from openedx.core.djangoapps.course_groups.models import CourseUserGroup
+
+
+class Command(BaseCommand):
+    """
+    Remove users with multiple cohorts of a course from all cohorts
+    """
+    help = 'Remove all users from multiple cohorts (except one) of each course'
+
+    def handle(self, *args, **options):
+        """
+        Execute the command
+        """
+        # Get entries of cohorts which have same user added multiple times for a single course
+        multiple_objects_cohorts = CourseUserGroup.objects.filter(group_type=CourseUserGroup.COHORT).\
+            values_list('users', 'course_id').annotate(user_count=Count('users')).filter(user_count__gt=1).\
+            order_by('users')
+        multiple_objects_cohorts_count = multiple_objects_cohorts.count()
+        multiple_course_cohorts_users = set(multiple_objects_cohorts.values_list('users', flat=True))
+        users_failed_to_cleanup = []
+
+        for user in User.objects.filter(id__in=multiple_course_cohorts_users):
+            print u"Removing user with id '{0}' from cohort groups".format(user.id)
+            try:
+                # remove user from only cohorts
+                user.course_groups.remove(*user.course_groups.filter(group_type=CourseUserGroup.COHORT))
+            except Exception as err:    # pylint: disable=broad-except
```

Comment:
any particular type of Exception expected here?  would the delete fail for any other reason than user.course_groups being empty?